### ISSUE TITLE
Canot Login and Register

### ISSUE THREAD

**Issue by rootbakar** [JanoCodes/Ticketing#160]
Hi team,, after i'm build this JanoCodes in my server,, and i access http://localhost/#login-modal.
after i have registered this direct to http://lcaolhost/register but on screen display "404 Not Found" 
![image](https://user-images.githubusercontent.com/43517550/55783387-8017f400-5ad8-11e9-90ac-92e83628035c.png)

**Comment by mpgn** [comment]
Same :'(

**Comment by xluccianox** [comment]
Do you have mod_rewrite enabled in Apache?
Check:
https://laravel.com/docs/5.0/configuration#pretty-urls

**Comment by mpgn** [comment]
yes https://github.com/JanoCodes/Ticketing/blob/master/Dockerfile#L45
